src/init.cpp

```diff
@@ -752,13 +753,19 @@ void InitParameterInteraction()
 
     // disable walletbroadcast and whitelistalwaysrelay in blocksonly mode
```

Comment:
I guess there are three options:

* Silently accept both variations
* Ignore
* Add it to the blacklist https://github.com/gmaxwell/bitcoin/blob/325c725fb6205e38142914acb9ed1733d8482d46/src/init.cpp#L892-L903